src/current/_includes/common/tls-bad-cipher-warning.md

```diff
@@ -0,0 +1,9 @@
+{{site.data.alerts.callout_info}}
+CockroachDB clusters accept TLS 1.2 or TLS 1.3 security certificates for client authentication and encryption-in-flight. However, certain less-secure TLS 1.2 cipher suites are rejected, in accordance with the IETF's recommended cipher list defined in [RFC 8447](https://datatracker.ietf.org/doc/html/rfc8447).
+
+SQL clients that are more than five years old may fail to connect, if they do not support any cipher suites that CockroachDB considers to be secure, and hence supports.
+
+To allow connections using these deprecated cipher suites, and hence support these older clients, you can set the `COCKROACH_TLS_ENABLE_OLD_CIPHER_SUITES` environment variable to `true` for your `cockroach start` command. However, this is not recommended as these cipher suites are considered insecure.
+
+Refer to the code for the current list of supported cipher suites: [CockroachDB on GitHub: tls_ciphersuites.go](https://github.com/cockroachdb/cockroach/blob/master/pkg/security/tls_ciphersuites.go)
```

Comment:
Hmm.. I'm not sure if we should link to a code file in Github repo. Is there a precedent for it across the docs?

Reply:
I don't see why not, the code is open source. If the truth changes, it'll be recorded here, so this is the canonical reference. I'll ping the team for a consensus.

Reply:
Ok, removed the link and just have the list now.